24.md

```diff
@@ -0,0 +1,57 @@
+NIP-24
+======
+
+Protect the Unknown
+-------------------
+
+`draft` `optional` `client` `author:giszmo`
+
+## Replaceable Events
+
+A *replaceable event* as defined in [NIP-01](01.md) and [NIP-16](16.md).
+
+## Client Behavior
+
+Clients MUST NOT change or delete Data they do not understand without user
+confirmation.
+
+Rationale
+---------
+
+### Different extensions to a feature
+
+Clients differ in their feature sets and as nips expand kinds, clients that do
+not consider unsupported parameters remove those when "replacing" events,
+breaking the clients' compatibility.
+
+Alice uses Branle at home and Damus on mobile. She configured `nip05` on Branle
+but when storing her `screen_name` on Damus using the kind `0` metadata, her
+`nip05` configuration disappeard. When configuring this on Damus again, her
```

Comment:
this phrasing confuses me. Is Damus the intended client here again? by configuring what again on Damus makes her loose the screen_name? is it supposed to be Branle again instead?